pkgs/os-specific/linux/kvdo/default.nix

```diff
@@ -30,6 +30,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/dm-vdo/kvdo";
     description = "A pair of kernel modules which provide pools of deduplicated and/or compressed block storage";
     platforms = platforms.linux;
-    broken = kernel.kernelOlder "5.15";
+    broken = kernel.kernelOlder "5.17";
```

Comment:
Could probably also add a `kernelNewer`, but I'm not sure for which release.

Reply:
builds on 6.6, broken on 6.8